cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -109,133 +76,255 @@ spec:
                       type: integer
                     manageMachineDisruptionBudgets:
                       type: boolean
-                useAllNodes:
+                skipUpgradeChecks:
                   type: boolean
-                nodes:
-                  items:
-                    properties:
-                      name:
-                        type: string
-                      config:
+                continueUpgradeAfterChecksEvenIfNotHealthy:
+                  type: boolean
+                mon:
+                  type: object
+                  properties:
+                    allowMultiplePerNode:
+                      type: boolean
+                    count:
+                      maximum: 9
+                      minimum: 0
+                      type: integer
+                    volumeClaimTemplate:
+                      type: object
+                      x-kubernetes-preserve-unknown-fields: true
```

Comment:
Don't we need to specify this at a higher level or in more places? For example, I thought all the `resources`, `placement`, and `healthCheck` types would need this. Otherwise, it disallows any properties specified, right?

Reply:
Based on the errors I've hit in testing today, I'm concerned we'll be breaking older schema. Can we use this field more frequently to allow unknown fields?

Reply:
Okay, I tried adding this field (`unknown...`) everywhere I see `type: object` and we don't have the schema for that.  Maybe that will minimize the errors you have been hitting. ;)